sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+The SIGs should strive to present the TOC with easily understandable and votable "propositions", each of which is supported by clear written evidence.  A proposition may be “to approve this project for incubation based on this [written ](https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md)[due diligence](https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md)[ investigation](https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md)”, or “to approve this landscape document based on these clear goals and evidence that it achieves them”.  It is of utmost importance that the information and proposals provided to the TOC by SIGs be highly accurate and unbiased, driven by the goal to improve the CNCF as a whole, rather than benefit one project or company over another.  We believe that the rising tide lifts all boats, and that is our goal.
+
+Key ideas here:
+
+* The TOC is the arbiter & editor and may always intervene and overrule 
+
+* The SIGs are the productive talent, and respected as such.
+
+SIGs may choose to spawn focussed and time-limited working groups to achieve some of their responsibilities (for example, to produce a specific educational white paper, or portfolio gap analysis report).  Working groups should have a clearly documented charter, timeline (typically a few quarters at most), and set of deliverables. Once the timeline has elapsed, or the deliverables delivered, the working group dissolves, or is explicitly re-chartered.
+
+### Specific SIG Responsibilities 
+
+#### Project Handling:
+
+    * Understand and document a high level roadmap of projects within this space, including CNCF and non-CNCF projects. Identify gaps in project landscape.
+
+    * For projects that fall within the CNCF, perform health checks.
+
+    * Perform discovery of and outreach to candidate projects
+
+    * Help candidate projects prepare for presentation to the TOC
+
+    * Every CNCF project will be assigned to one suitable SIG by the TOC.
+
+#### End User Education (Outbound Communication)
+
+    * Provide up-to-date, high quality, unbiased and easy-to-consume material to help end users to understand and effectively adopt cloud-native technologies and practises within the SIG’s area, for example:
+
+        * White papers, presentations, videos, or other forms of training clarifying terminology, comparisons of different approaches, available projects or products, common or recommended practises, trends, illustrative successes and failures, etc.
```

Comment:
Some WG/Project teams provide white papers, presentations, etc.  What is the difference of SIG ones from those produced by the WG/Project teams?

Reply:
As previously discussed by Alexis, existing WG's will be shut down and reconstituted under the SIG structure. See above wording:

"SIGs may choose to spawn focussed and time-limited working groups to achieve some of their responsibilities (for example, to produce a specific educational white paper, or portfolio gap analysis report).  Working groups should have a clearly documented charter, timeline (typically a few quarters at most), and set of deliverables. Once the timeline has elapsed, or the deliverables delivered, the working group dissolves, or is explicitly re-chartered."